fbw-common/src/systems/instruments/src/EFB/Apis/Simbrief/ofpLayouts.json

```diff
@@ -0,0 +1,174 @@
+        "Maps"
+    ],
+    "BAW": [
+        "All",
+        "Summary",
+        "Weights and Performance",
+        "Primary Nav Log",
+        "Alternate Nav Log",
+        "Wind Info",
+        "ATC Flight Plan",
+        "Dispatch Information",
+        "Weather",
+        "NOTAM",
+        "Crew Information",
+        "Maps",
+        "Additional Docs",
+        "RAIM",
+        "Additional Briefing Documents",
+        "Runway Analysis"
+    ],
+    "DAL": [
+        "All",
+        "Release",
+        "Plan Summary",
+        "Plan Fuel",
+        "DX Remarks",
+        "Flight Log",
+        "Enroute Winds",
+        "NOTAMs and Remarks",
+        "Runway Analysis",
+        "Maps"
+    ],
+    "DLH": [
+        "All",
+        "OFP",
+        "ATC Flight Plan",
+        "Dispatch Information",
+        "Crew Information",
+        "Weather",
+        "NOTAM",
+        "Maps",
+        "Official VAA/TCA",
+        "CDA",
+        "Additional Docs",
+        "Dispatcher Maps",
+        "RAIM",
+        "Runway Analysis"
+    ],
+    "EIN": [
+        "All",
+        "Summary and Times",
+        "Fuel and Weights",
+        "Flight Log",
+        "Waypoints",
+        "ATC Flight Plan",
+        "Dispatch Information",
+        "Weather",
+        "NOTAM",
+        "Crew Information",
+        "Maps",
+        "Official VAA/TCA",
+        "Additional Docs",
+        "RAIM",
+        "Additional Briefing Documents",
+        "Runway Analysis"
+    ],
+    "EZY": [
+        "All",
+        "Summary and Fuel",
+        "Flight Log",
+        "Airport Data",
+        "Wind Information",
+        "ATC Flight Plan",
+        "Runway Analysis",
+        "Airport WX List",
+        "NOTAM",
+        "Company NOTAM",
+        "Maps"
+    ],
+    "JBU": [
+        "All",
+        "Acceptance",
+        "Dispatch Release",
+        "Flight Plan",
+        "Airport Briefing",
+        "Runway Analysis",
+        "Maps"
+    ],
+    "QFA": [
+        "All",
+        "Computer Flight Plan",
+        "Weather and NOTAMs",
+        "Runway Analysis",
+        "Maps"
+    ],
+    "THY": [
+        "All",
+        "OFP",
+        "ATC Flight Plan",
+        "Additional Info",
+        "Runway Analysis",
+        "Airport WX List",
+        "NOTAM",
+        "Company NOTAM",
+        "Maps"
+    ],
+    "KLM": [
+        "All",
+        "Dispatch Briefing",
+        "Administrator Notes",
+        "OFP Flight Particulars",
+        "OFP Navigation Part",
+        "OFP Alternate Summary",
+        "Air Traffic Control",
+        "Lido Briefing",
+        "Runway Analysis",
+        "UAD Charts",
+        "Airports Weather",
+        "NOTAM",
+        "Crew Alert"
+    ],
+    "BER": [
+        "All",
+        "Summary",
+        "Flight Log",
+        "Wind Information",
+        "ATC Flight Plan",
+        "Additional Info",
+        "Runway Analysis",
+        "Weather",
+        "Notam",
+        "Crew Alert & Bulletin",
+        "Maps"
+    ]
+}
```

Comment:
Long term I'd like to move this into either base/config or as part of aircraft context (I've not decided yet which would fit better for this). Having loose .json around the repository is hard for maintainers to keep track of.